libs/wallet-provider/README.md

```diff
@@ -0,0 +1,25 @@
+# CoW Swap Wallet provider
+
```

Comment:
nit, would be nice a one-liner here explaining what is this for (clarifies the scope of the lib)

Reply:
Thanks! Added description: https://github.com/cowprotocol/cowswap/pull/4414/commits/1e3012ac027ecc2c792fce89adb7c6245be4e55c